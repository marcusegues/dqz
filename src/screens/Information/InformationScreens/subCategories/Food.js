// @flow
import React from 'react';
// $FlowFixMe
import { TouchableOpacity } from 'react-native';
import type { ComponentType } from 'react';
import { translate } from 'react-i18next';
import type { Navigation, TFunction } from '../../../../types/generalTypes';
import { verticalScale } from '../../../../styles/Scaling';
import { SubCategoryContainer } from '../../subComponents/SubCategoryContainer';
import { AppInfoSubText } from '../../../AppInfo/subComponents/AppInfoSubText';
import { CardRowText } from '../../../../components/QuestionAnswer/Cards/subcomponents/CardRowText';

type FoodInnerProps = {
  t: TFunction,
  navigation: Navigation,
};

const FoodInner = ({ t, navigation }: FoodInnerProps) => (
  <SubCategoryContainer title={t('foodMainText')}>
    <AppInfoSubText
      text={t('foodText1')}
      style={{ marginTop: verticalScale(10) }}
    />
    <AppInfoSubText
      text={t('foodText2')}
      style={{ marginTop: verticalScale(10) }}
    />
    <AppInfoSubText
      text={t('foodText3')}
      style={{ marginTop: verticalScale(10) }}
    />
    <CardRowText
      text={t('foodText4')}
      style={{ marginTop: verticalScale(20) }}
    />
    <AppInfoSubText
      text={t('foodText5')}
      style={{ marginTop: verticalScale(10) }}
    />
    <CardRowText
      text={t('foodText6')}
      style={{ marginTop: verticalScale(20) }}
    />
    <AppInfoSubText
      text={t('foodText7')}
      style={{ marginTop: verticalScale(10) }}
    />
    <TouchableOpacity onPress={() => navigation.navigate('VatAllowance')}>
      <AppInfoSubText
        text={t('foodText8')}
        style={{ marginTop: verticalScale(10) }}
      />
    </TouchableOpacity>
    <TouchableOpacity onPress={() => navigation.navigate('DutyAllowance')}>
      <AppInfoSubText
        text={t('foodText9')}
        style={{ marginTop: verticalScale(10) }}
      />
    </TouchableOpacity>
  </SubCategoryContainer>
);

export const Food = (translate(['food'])(FoodInner): ComponentType<
  FoodInnerProps
>);
