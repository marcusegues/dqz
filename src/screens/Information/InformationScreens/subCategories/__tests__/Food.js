import React from 'react';
// $FlowFixMe
import { TouchableOpacity } from 'react-native';
import { shallow } from 'enzyme';
import { Food } from '../Food';

jest.mock('react-i18next', () => ({
  translate: () => Component => props => <Component t={() => ''} {...props} />,
}));

const clickFn = jest.fn();

describe('Food', () => {
  test('Food renders as expected', () => {
    const wrapper = shallow(<Food />);
    expect(wrapper).toMatchSnapshot();
  });
  test('Food renders as expected with language', () => {
    const wrapper = shallow(
      <Food navigation={{ dispatch: clickFn }} i18n={{ language: 'de' }} />
    );
    expect(wrapper).toMatchSnapshot();
  });
  test('Food renders as expected with language with dive()', () => {
    const wrapper = shallow(
      <Food
        navigation={{ dispatch: clickFn }}
        i18n={{ language: 'de', t: 'test' }}
      />
    );
    expect(wrapper.dive()).toMatchSnapshot();
  });
  test('Food renders as expected with language with dive()', () => {
    const wrapper = shallow(
      <Food
        navigation={{ dispatch: clickFn }}
        i18n={{ language: 'de', t: 'test' }}
      />
    );
    const render = wrapper.dive();
    render.find(TouchableOpacity).forEach(child => {
      child.simulate('press');
    });
    expect(wrapper.dive()).toMatchSnapshot();
    expect(clickFn.mock.calls.length).toBe(2);
  });
});
