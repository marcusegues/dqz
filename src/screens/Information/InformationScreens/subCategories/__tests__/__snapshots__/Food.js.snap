// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Food Food renders as expected 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <FoodInner
    navigation={
      Object {
        "dispatch": [Function],
      }
    }
    t={[Function]}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <AppInfoSubText
          greyBox={false}
          style={
            Object {
              "marginTop": 19.61764705882353,
            }
          }
          text=""
        />,
        <AppInfoSubText
          greyBox={false}
          style={
            Object {
              "marginTop": 19.61764705882353,
            }
          }
          text=""
        />,
        <AppInfoLink
          onPress={[Function]}
          style={
            Object {
              "marginTop": 19.61764705882353,
            }
          }
          text=""
        />,
        <CardRowText
          greyBox={false}
          style={
            Object {
              "marginTop": 39.23529411764706,
            }
          }
          text=""
        />,
        <AppInfoSubText
          greyBox={false}
          style={
            Object {
              "marginTop": 19.61764705882353,
            }
          }
          text=""
        />,
        <CardRowText
          greyBox={false}
          style={
            Object {
              "marginTop": 39.23529411764706,
            }
          }
          text=""
        />,
        <AppInfoSubText
          greyBox={false}
          style={
            Object {
              "marginTop": 19.61764705882353,
            }
          }
          text=""
        />,
        <TouchableOpacity
          activeOpacity={0.2}
          onPress={[Function]}
        >
          <AppInfoSubText
            greyBox={false}
            style={
              Object {
                "marginTop": 19.61764705882353,
                "textDecorationLine": "underline",
                "textDecorationStyle": "solid",
              }
            }
            text=""
          />
        </TouchableOpacity>,
        <TouchableOpacity
          activeOpacity={0.2}
          onPress={[Function]}
        >
          <AppInfoSubText
            greyBox={false}
            style={
              Object {
                "marginTop": 19.61764705882353,
                "textDecorationLine": "underline",
                "textDecorationStyle": "solid",
              }
            }
            text=""
          />
        </TouchableOpacity>,
      ],
      "title": "",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "greyBox": false,
          "style": Object {
            "marginTop": 19.61764705882353,
          },
          "text": "",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "greyBox": false,
          "style": Object {
            "marginTop": 19.61764705882353,
          },
          "text": "",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "onPress": [Function],
          "style": Object {
            "marginTop": 19.61764705882353,
          },
          "text": "",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "greyBox": false,
          "style": Object {
            "marginTop": 39.23529411764706,
          },
          "text": "",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "greyBox": false,
          "style": Object {
            "marginTop": 19.61764705882353,
          },
          "text": "",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "greyBox": false,
          "style": Object {
            "marginTop": 39.23529411764706,
          },
          "text": "",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "greyBox": false,
          "style": Object {
            "marginTop": 19.61764705882353,
          },
          "text": "",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "activeOpacity": 0.2,
          "children": <AppInfoSubText
            greyBox={false}
            style={
              Object {
                "marginTop": 19.61764705882353,
                "textDecorationLine": "underline",
                "textDecorationStyle": "solid",
              }
            }
            text=""
          />,
          "onPress": [Function],
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "greyBox": false,
            "style": Object {
              "marginTop": 19.61764705882353,
              "textDecorationLine": "underline",
              "textDecorationStyle": "solid",
            },
            "text": "",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "activeOpacity": 0.2,
          "children": <AppInfoSubText
            greyBox={false}
            style={
              Object {
                "marginTop": 19.61764705882353,
                "textDecorationLine": "underline",
                "textDecorationStyle": "solid",
              }
            }
            text=""
          />,
          "onPress": [Function],
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "greyBox": false,
            "style": Object {
              "marginTop": 19.61764705882353,
              "textDecorationLine": "underline",
              "textDecorationStyle": "solid",
            },
            "text": "",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <AppInfoSubText
            greyBox={false}
            style={
              Object {
                "marginTop": 19.61764705882353,
              }
            }
            text=""
          />,
          <AppInfoSubText
            greyBox={false}
            style={
              Object {
                "marginTop": 19.61764705882353,
              }
            }
            text=""
          />,
          <AppInfoLink
            onPress={[Function]}
            style={
              Object {
                "marginTop": 19.61764705882353,
              }
            }
            text=""
          />,
          <CardRowText
            greyBox={false}
            style={
              Object {
                "marginTop": 39.23529411764706,
              }
            }
            text=""
          />,
          <AppInfoSubText
            greyBox={false}
            style={
              Object {
                "marginTop": 19.61764705882353,
              }
            }
            text=""
          />,
          <CardRowText
            greyBox={false}
            style={
              Object {
                "marginTop": 39.23529411764706,
              }
            }
            text=""
          />,
          <AppInfoSubText
            greyBox={false}
            style={
              Object {
                "marginTop": 19.61764705882353,
              }
            }
            text=""
          />,
          <TouchableOpacity
            activeOpacity={0.2}
            onPress={[Function]}
          >
            <AppInfoSubText
              greyBox={false}
              style={
                Object {
                  "marginTop": 19.61764705882353,
                  "textDecorationLine": "underline",
                  "textDecorationStyle": "solid",
                }
              }
              text=""
            />
          </TouchableOpacity>,
          <TouchableOpacity
            activeOpacity={0.2}
            onPress={[Function]}
          >
            <AppInfoSubText
              greyBox={false}
              style={
                Object {
                  "marginTop": 19.61764705882353,
                  "textDecorationLine": "underline",
                  "textDecorationStyle": "solid",
                }
              }
              text=""
            />
          </TouchableOpacity>,
        ],
        "title": "",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "greyBox": false,
            "style": Object {
              "marginTop": 19.61764705882353,
            },
            "text": "",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "greyBox": false,
            "style": Object {
              "marginTop": 19.61764705882353,
            },
            "text": "",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "onPress": [Function],
            "style": Object {
              "marginTop": 19.61764705882353,
            },
            "text": "",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "greyBox": false,
            "style": Object {
              "marginTop": 39.23529411764706,
            },
            "text": "",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "greyBox": false,
            "style": Object {
              "marginTop": 19.61764705882353,
            },
            "text": "",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "greyBox": false,
            "style": Object {
              "marginTop": 39.23529411764706,
            },
            "text": "",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "greyBox": false,
            "style": Object {
              "marginTop": 19.61764705882353,
            },
            "text": "",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "activeOpacity": 0.2,
            "children": <AppInfoSubText
              greyBox={false}
              style={
                Object {
                  "marginTop": 19.61764705882353,
                  "textDecorationLine": "underline",
                  "textDecorationStyle": "solid",
                }
              }
              text=""
            />,
            "onPress": [Function],
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "greyBox": false,
              "style": Object {
                "marginTop": 19.61764705882353,
                "textDecorationLine": "underline",
                "textDecorationStyle": "solid",
              },
              "text": "",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "activeOpacity": 0.2,
            "children": <AppInfoSubText
              greyBox={false}
              style={
                Object {
                  "marginTop": 19.61764705882353,
                  "textDecorationLine": "underline",
                  "textDecorationStyle": "solid",
                }
              }
              text=""
            />,
            "onPress": [Function],
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "greyBox": false,
              "style": Object {
                "marginTop": 19.61764705882353,
                "textDecorationLine": "underline",
                "textDecorationStyle": "solid",
              },
              "text": "",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`Food Food renders as expected with language with dive() 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <FoodInner
    navigation={
      Object {
        "dispatch": [Function],
      }
    }
    t={[Function]}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <AppInfoSubText
          greyBox={false}
          style={
            Object {
              "marginTop": 19.61764705882353,
            }
          }
          text=""
        />,
        <AppInfoSubText
          greyBox={false}
          style={
            Object {
              "marginTop": 19.61764705882353,
            }
          }
          text=""
        />,
        <AppInfoLink
          onPress={[Function]}
          style={
            Object {
              "marginTop": 19.61764705882353,
            }
          }
          text=""
        />,
        <CardRowText
          greyBox={false}
          style={
            Object {
              "marginTop": 39.23529411764706,
            }
          }
          text=""
        />,
        <AppInfoSubText
          greyBox={false}
          style={
            Object {
              "marginTop": 19.61764705882353,
            }
          }
          text=""
        />,
        <CardRowText
          greyBox={false}
          style={
            Object {
              "marginTop": 39.23529411764706,
            }
          }
          text=""
        />,
        <AppInfoSubText
          greyBox={false}
          style={
            Object {
              "marginTop": 19.61764705882353,
            }
          }
          text=""
        />,
        <TouchableOpacity
          activeOpacity={0.2}
          onPress={[Function]}
        >
          <AppInfoSubText
            greyBox={false}
            style={
              Object {
                "marginTop": 19.61764705882353,
                "textDecorationLine": "underline",
                "textDecorationStyle": "solid",
              }
            }
            text=""
          />
        </TouchableOpacity>,
        <TouchableOpacity
          activeOpacity={0.2}
          onPress={[Function]}
        >
          <AppInfoSubText
            greyBox={false}
            style={
              Object {
                "marginTop": 19.61764705882353,
                "textDecorationLine": "underline",
                "textDecorationStyle": "solid",
              }
            }
            text=""
          />
        </TouchableOpacity>,
      ],
      "title": "",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "greyBox": false,
          "style": Object {
            "marginTop": 19.61764705882353,
          },
          "text": "",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "greyBox": false,
          "style": Object {
            "marginTop": 19.61764705882353,
          },
          "text": "",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "onPress": [Function],
          "style": Object {
            "marginTop": 19.61764705882353,
          },
          "text": "",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "greyBox": false,
          "style": Object {
            "marginTop": 39.23529411764706,
          },
          "text": "",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "greyBox": false,
          "style": Object {
            "marginTop": 19.61764705882353,
          },
          "text": "",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "greyBox": false,
          "style": Object {
            "marginTop": 39.23529411764706,
          },
          "text": "",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "greyBox": false,
          "style": Object {
            "marginTop": 19.61764705882353,
          },
          "text": "",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "activeOpacity": 0.2,
          "children": <AppInfoSubText
            greyBox={false}
            style={
              Object {
                "marginTop": 19.61764705882353,
                "textDecorationLine": "underline",
                "textDecorationStyle": "solid",
              }
            }
            text=""
          />,
          "onPress": [Function],
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "greyBox": false,
            "style": Object {
              "marginTop": 19.61764705882353,
              "textDecorationLine": "underline",
              "textDecorationStyle": "solid",
            },
            "text": "",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "activeOpacity": 0.2,
          "children": <AppInfoSubText
            greyBox={false}
            style={
              Object {
                "marginTop": 19.61764705882353,
                "textDecorationLine": "underline",
                "textDecorationStyle": "solid",
              }
            }
            text=""
          />,
          "onPress": [Function],
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "greyBox": false,
            "style": Object {
              "marginTop": 19.61764705882353,
              "textDecorationLine": "underline",
              "textDecorationStyle": "solid",
            },
            "text": "",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <AppInfoSubText
            greyBox={false}
            style={
              Object {
                "marginTop": 19.61764705882353,
              }
            }
            text=""
          />,
          <AppInfoSubText
            greyBox={false}
            style={
              Object {
                "marginTop": 19.61764705882353,
              }
            }
            text=""
          />,
          <AppInfoLink
            onPress={[Function]}
            style={
              Object {
                "marginTop": 19.61764705882353,
              }
            }
            text=""
          />,
          <CardRowText
            greyBox={false}
            style={
              Object {
                "marginTop": 39.23529411764706,
              }
            }
            text=""
          />,
          <AppInfoSubText
            greyBox={false}
            style={
              Object {
                "marginTop": 19.61764705882353,
              }
            }
            text=""
          />,
          <CardRowText
            greyBox={false}
            style={
              Object {
                "marginTop": 39.23529411764706,
              }
            }
            text=""
          />,
          <AppInfoSubText
            greyBox={false}
            style={
              Object {
                "marginTop": 19.61764705882353,
              }
            }
            text=""
          />,
          <TouchableOpacity
            activeOpacity={0.2}
            onPress={[Function]}
          >
            <AppInfoSubText
              greyBox={false}
              style={
                Object {
                  "marginTop": 19.61764705882353,
                  "textDecorationLine": "underline",
                  "textDecorationStyle": "solid",
                }
              }
              text=""
            />
          </TouchableOpacity>,
          <TouchableOpacity
            activeOpacity={0.2}
            onPress={[Function]}
          >
            <AppInfoSubText
              greyBox={false}
              style={
                Object {
                  "marginTop": 19.61764705882353,
                  "textDecorationLine": "underline",
                  "textDecorationStyle": "solid",
                }
              }
              text=""
            />
          </TouchableOpacity>,
        ],
        "title": "",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "greyBox": false,
            "style": Object {
              "marginTop": 19.61764705882353,
            },
            "text": "",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "greyBox": false,
            "style": Object {
              "marginTop": 19.61764705882353,
            },
            "text": "",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "onPress": [Function],
            "style": Object {
              "marginTop": 19.61764705882353,
            },
            "text": "",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "greyBox": false,
            "style": Object {
              "marginTop": 39.23529411764706,
            },
            "text": "",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "greyBox": false,
            "style": Object {
              "marginTop": 19.61764705882353,
            },
            "text": "",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "greyBox": false,
            "style": Object {
              "marginTop": 39.23529411764706,
            },
            "text": "",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "greyBox": false,
            "style": Object {
              "marginTop": 19.61764705882353,
            },
            "text": "",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "activeOpacity": 0.2,
            "children": <AppInfoSubText
              greyBox={false}
              style={
                Object {
                  "marginTop": 19.61764705882353,
                  "textDecorationLine": "underline",
                  "textDecorationStyle": "solid",
                }
              }
              text=""
            />,
            "onPress": [Function],
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "greyBox": false,
              "style": Object {
                "marginTop": 19.61764705882353,
                "textDecorationLine": "underline",
                "textDecorationStyle": "solid",
              },
              "text": "",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "activeOpacity": 0.2,
            "children": <AppInfoSubText
              greyBox={false}
              style={
                Object {
                  "marginTop": 19.61764705882353,
                  "textDecorationLine": "underline",
                  "textDecorationStyle": "solid",
                }
              }
              text=""
            />,
            "onPress": [Function],
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "greyBox": false,
              "style": Object {
                "marginTop": 19.61764705882353,
                "textDecorationLine": "underline",
                "textDecorationStyle": "solid",
              },
              "text": "",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
